### DISCUSSION TITLE
how element cleanup be skipped

### DISCUSSION THREAD

**Discussion by luovn** [tinymce/tinymce#9272] [General]
Using version V6.7.2,   I tried to insert the following code and it will be automatically cleared

`<script` type="text/javascript">$('#new').html("<div>News</div>")</script>
![QQ图片2](https://github.com/tinymce/tinymce/assets/28971780/c6b05390-55fc-4137-b7b3-0f86269b393a)

**Comment by lostkeys** [comment]
Hi @luovn! That is by design. Please learn more about how this works  in our [security guide](https://www.tiny.cloud/docs/tinymce/6/security/).
— Fredrik, Product Manager

**Reply by luovn** [reply]
Is there a way to avoid filtering and save it

**Reply by TheSpyder** [reply]
It's covered in the guide
https://www.tiny.cloud/docs/tinymce/6/security/#turning-dompurify-off
